imap_processing/codice/codice_l1a.py

```diff
@@ -104,6 +105,11 @@ def create_science_dataset(self, start_time: np.datetime64) -> xr.Dataset:
         xr.Dataset
             ``xarray`` dataset containing the science data and supporting metadata
         """
+        cdf_attrs = ImapCdfAttributes()
+        cdf_attrs.add_instrument_global_attrs("codice")
+        cdf_attrs.add_instrument_variable_attrs("codice", "l1a")
```

Comment:
It's great to see how easy of transition it is. I was concerned about that! Nice job @maxinelasp and all!

Reply:
Agreed! I thought it was really straightforward!